Bugzilla/Install/Localconfig.pm

```diff
@@ -141,6 +141,8 @@ use constant LOCALCONFIG_VARS => (
             shadowdbhost         => undef,
             shadowdbport         => undef,
             shadowdbsock         => undef,
+            shadowdbuser         => undef,
```

Comment:
instead of having these in the override thing, just add them directly above (e.g. the same as "urlbase" or "apache_size_limit"